SMLHelper/Patchers/CustomFishPatcher.cs

```diff
@@ -0,0 +1,68 @@
+﻿
+
+namespace SMLHelper.V2.FishFramework
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Reflection;
+    using Harmony;
+    using UnityEngine;
+    using Random = UnityEngine.Random;
+    using SMLHelper.V2.Handlers;
+    using Logger = SMLHelper.V2.Logger;
+
+    internal class CustomFishPatcher
+    {
+        static List<Creature> usedCreatures = new List<Creature>();
+
+        private static void CreatureStart_Postfix(Creature __instance)
+        {
+            if(usedCreatures.Contains(__instance) || CustomFishHandler.fishTechTypes.Count == 0)
+            {
+                return;
+            }
+            TechTag tag = __instance.GetComponent<TechTag>();
+            if(tag)
+            {
+                if(CustomFishHandler.fishTechTypes.Contains(tag.type))
+                {
+                    return;
+                }
+            }
+            if(Random.value < 0.1f)
+            {
+                Console.WriteLine($"[FishFramework] Selecting fish out of {CustomFishHandler.fishTechTypes.Count} total types");
```

Comment:
Can these Console writes be updated to use our [Logger](https://github.com/SMLHelper/SMLHelper/blob/2.x/SMLHelper/Logger.cs) class?  
Regarding the logging levels:  
- Info = "Always printed"
- Warn = "Only printed for recoverable errors that _may_ result in unintended behavior"
- Error = "Only printed for critical errors that _absolutely_ will result in unintended behavior"
- Debug = "Only printed when debug logging in enabled. Used for things like the normal execution of patches."